### PR TITLE
Fixed testing instructions

### PR DESCRIPTION
**Pull request by octaedro**
This PR updates the testing instructions.

After having a few new [WC Admin Test Helper](https://github.com/woocommerce/woocommerce-admin-test-helper) features in place, and its wiki updated ( [#](https://github.com/woocommerce/woocommerce-admin-test-helper/wiki/Update-option) [#](https://github.com/woocommerce/woocommerce-admin-test-helper/wiki/Admin-notifications) [#](https://github.com/woocommerce/woocommerce-admin-test-helper/wiki/Tools) ), the [testing instructions](https://github.com/woocommerce/woocommerce-admin/blob/main/TESTING-INSTRUCTIONS.md) needed a few changes.
This PR applies those changes to the file in order to write down the new instructions.

No changelog necessary.

### Detailed test instructions:

-   Verify everything looks good in the file

### CODE REVIEW

File: TESTING-INSTRUCTIONS.md [hunk 1]
```diff
@@ -268,12 +269,10 @@ In case the report shows "no data", please reimport historical data by following
 
 -   Create a brand new site.
 -   Install a plugin to log every sent email (you can use [WP mail logging](https://wordpress.org/plugins/wp-mail-logging/)).
--   Install and active [this gist](https://gist.github.com/octaedro/864315edaf9c6a2a6de71d297be1ed88) to create an email note. Just download the file and install it as a plugin.
--   After activating the plugin, press `Add Email Notes` to create a note.
+-   Install and activate [WC Admin Test Helper](https://github.com/woocommerce/woocommerce-admin-test-helper/wiki/Admin-notifications).
+-   Create a [new note type `email`](https://github.com/woocommerce/woocommerce-admin-test-helper/wiki/Admin-notifications).
```
Comment (ilyasfoo) [inline TESTING-INSTRUCTIONS.md:273]:
```suggestion
-   Create a [new note type `email`](https://github.com/woocommerce/woocommerce-admin-test-helper/wiki/Admin-notifications#3---email).
```

Maybe you meant to link directly to email?

File: TESTING-INSTRUCTIONS.md [hunk 2]
```diff
@@ -268,12 +269,10 @@ In case the report shows "no data", please reimport historical data by following
 
 -   Create a brand new site.
 -   Install a plugin to log every sent email (you can use [WP mail logging](https://wordpress.org/plugins/wp-mail-logging/)).
--   Install and active [this gist](https://gist.github.com/octaedro/864315edaf9c6a2a6de71d297be1ed88) to create an email note. Just download the file and install it as a plugin.
--   After activating the plugin, press `Add Email Notes` to create a note.
+-   Install and activate [WC Admin Test Helper](https://github.com/woocommerce/woocommerce-admin-test-helper/wiki/Admin-notifications).
```
Comment (ilyasfoo) [inline TESTING-INSTRUCTIONS.md:272]:
```suggestion
-   Install and activate [WC Admin Test Helper](https://github.com/woocommerce/woocommerce-admin-test-helper/releases).
```

I'm thinking a link to download the latest release is a shorter path. There are a couple more links like this below, if you agree to change search and replace would be quicker

File: TESTING-INSTRUCTIONS.md [hunk 3]
```diff
@@ -439,7 +438,8 @@ wp.hooks.addFilter( 'woocommerce_admin_profile_wizard_steps', 'woocommerce-admin
 UPDATE `wp_options` SET `option_value`=UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL 5 day)) WHERE `option_name` = 'woocommerce_admin_install_timestamp';
 ```
 
--   Run the cron (this tool can help [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)).
+-   Install and activate [WC Admin Test Helper](https://github.com/woocommerce/woocommerce-admin-test-helper/wiki/Admin-notifications).
```
Comment (ilyasfoo) [inline TESTING-INSTRUCTIONS.md:441]:
```suggestion
-   Install and activate [WC Admin Test Helper](https://github.com/woocommerce/woocommerce-admin-test-helper/releases).
```

**Review by ilyasfoo — APPROVED** [review]
Looking good, just a couple of minor comment for improving the links. Pre-approving!

File: TESTING-INSTRUCTIONS.md (hunk 2 above)
Comment (octaedro) [reply TESTING-INSTRUCTIONS.md:272]: Good catch!

File: TESTING-INSTRUCTIONS.md (hunk 1 above)
Comment (octaedro) [reply TESTING-INSTRUCTIONS.md:273]: Good idea 👍

**Pull request merged**